grc/core/blocks/block.py

```diff
@@ -151,7 +151,8 @@ def rekey(ports):
             # this is ok behavior, unfortunately we could be hiding other import bugs
             pass
         except Exception:
-            self.add_error_message(f'Failed to evaluate import expression {imports!r}')
+            log.exception('Failed to evaluate import expression "{0}"'.format(expr), exc_info=True)
```

Comment:
a change to 
```python
self.add_error_message("Failed to evaluate import expression {!r}".format(imports))
```
should re-enable legacy Python support. Python 2.6 would require an additional 0 in the curly braces